drivers/serial/arm/uart.c

```diff
@@ -17,6 +16,18 @@
 #define LOG_DRIVER(...) do{ microkit_dbg_puts(microkit_name); microkit_dbg_puts("|INFO: "); microkit_dbg_puts(__VA_ARGS__); }while(0)
 #define LOG_DRIVER_ERR(...) do{ microkit_dbg_puts(microkit_name); microkit_dbg_puts("|ERROR: "); microkit_dbg_puts(__VA_ARGS__); }while(0)
 
+static void assert_fail(const char *s, int line)
```

Comment:
Don't think we should be doing this in a driver.

Reply:
Looks like this has been fixed elsewhere.